spec/system/legislation/processes_spec.rb

```diff
@@ -127,6 +127,19 @@
         expect(page).to have_content("past published")
       end
     end
+
+    scenario "Show SDG tags when feature is enabled" do
+      Setting["feature.sdg"] = true
+      Setting["sdg.process.legislation"] = true
+
+      create(:legislation_process, sdg_goals: [SDG::Goal[1]],
+                                   sdg_targets: [SDG::Target["1.1"]])
+
+      visit legislation_processes_path
+
+      expect(page).to have_selector(".sdg-goal-tag-list")
+      expect(page).to have_selector(".sdg-target-tag-list")
```

Comment:
How about testing it from the user's point of view?

```suggestion
      expect(page).to have_selector "img[alt='1. No Poverty']"
      expect(page).to have_content "target 1.1"
```